player/models.py

```diff
@@ -0,0 +1,15 @@
+from django.db import models
+from django.conf import settings
+from player.enums import BallGame
+
+
+class Player(models.Model):
+    user = models.OneToOneField(settings.AUTH_USER_MODEL,
+                                on_delete=models.CASCADE)
```

Comment:
should it have a player_id as well ?

Reply:
so it's mandatory to add an id field as well? 
but i thought that using the oneToOne User field will make it so the user fields will be added to the model.. guess i will conduct some more reserach

Reply:
I see that you found a solution by specifying the user as pk